plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/aps/system/services/kie/KieFormManager.java

```diff
@@ -400,22 +400,26 @@ public KieTaskDetail getTaskDetail(KieBpmConfig config, String containerId, fina
     }
 
     @Override
-    public String getProcInstDiagramImage(KieBpmConfig config, String containerId, String processId) throws ApsSystemException {
+    public String getProcInstDiagramImage(KieBpmConfig config, String containerId) throws ApsSystemException {
         return getProcInstDiagramImage(config,
-                 containerId, processId,
+                 containerId, 
                  null);
     }
 
     @Override
-    public String getProcInstDiagramImage(KieBpmConfig config, String containerId, String processId, Long pInstanceId) throws ApsSystemException {
+    public String getProcInstDiagramImage(KieBpmConfig config, String containerId,  String pInstanceId) throws ApsSystemException {
```

Comment:
Do we have sufficient test coverage on this method? I see no major structural changes but if we don't have a test lets add one now

Reply:
Nope, I'll create a new test for this method